README.md

```diff
@@ -1,37 +1,161 @@
+
+## Contents
+
+* [Before you start](#before-you-start)
+* [Getting Started](#getting-started)
+* [Stopping the Hub](#stopping-the-hub)
+* [Startup Commandline Options](#Startup-Commandline-Options)
+* [Useful Scripts](#useful-scripts)
+    * [XMLSecTool in Docker](#XMLSecTool-in-Docker)
+    * [Generate PKI](#Generate-PKI)
+    * [Create environment for local running](#Create-environment-for-local-running)
+* [Support and raising issues](#Support-and-raising-issues)
+* [Licence](#Licence)
 
 ## Before you start
 
-You'll need to install [cfssl](https://github.com/cloudflare/cfssl) which is used to generate the PKI.
+You'll need a computer with at least 8GB (for linux systems) or 16GB (for Mac and Windows systems) and at least a 4 core processor.  You'll also need administrator privilages in order to installed the required components listed below.
+
+You will need to install Git, [Docker](https://www.docker.com/get-started) and Ruby 2.7.2.  If you have [rbenv](https://github.com/rbenv/rbenv) installed the script will automatically pull and compile the correct version of ruby for you.
+
+Ideally if you are on a Linux based system you'll also need to add yourself to the docker group:
+
+```sh
+sudo usermod -a -G docker $(whoami)
+```
+
+### Notes for Mac Systems
+
+Docker does not run natively on Mac OS systems and instead uses a VirtualBox Virtual Machine which is controlled by the DockerApp.  Before you start you need to allocate at least 6GB of RAM (Ideally 8GB) to this virtual machine.  Additionally if you increase the number of CPUs allocated from the default of 2 you can improve the build performance.  Details can be found in the [Docker User Manual](https://docs.docker.com/docker-for-mac/#resources).
+
+## Getting Started
+
+To run the hub locally first you need to clone this repository to an empty directory.
+
+```sh
+mkdir Verify
+cd Verify
+git clone https://github.com/alphagov/verify-local-startup.git
+cd verify-local-startup
+```
+
+Now you're in the `verify-local-startup` directory running the hub should be as simple as:
+
+```sh
+./startup.sh
+```
+
+If your running on Linux we highly recommend you use the `-d` commandline option which additionally runs [Dozzle](https://dozzle.dev/) on port 50999.  Which allows you to see the logs for the various Hub microservices.
+
+```sh
+./startup.sh -d
+```
+ 
+If everything works as expected you should see the following message:
+
+![Screenshot of a successful build](docs/sucessful-build.png)
+
+> Started - visit [http://localhost:50130/test-rp](http://localhost:50130/test-rp) to start a journey (may take some time to spin up)
+
+It can take up to 5 minutes for the 13 indivual microservices to get going and for you to make a successful verify journey depending on the age and power of your computer.  If you encounter any issues please [raise an issue](#Support-and-raising-issues).
```

Comment:
```suggestion
It can take up to 5 minutes for the 13 individual microservices to get going and for you to make a successful verify journey depending on the age and power of your computer.  If you encounter any issues please [raise an issue](#Support-and-raising-issues).
```